android/expoview/src/main/java/host/exp/exponent/kernel/services/sensors/SensorKernelServiceSubscription.kt

```diff
@@ -0,0 +1,49 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+package host.exp.exponent.kernel.services.sensors
+
+import host.exp.exponent.kernel.ExperienceKey
+
+class SensorKernelServiceSubscription internal constructor(
+  val experienceKey: ExperienceKey,
+  private val subscribableSensorKernelService: SubscribableSensorKernelService,
+  val sensorEventListener: SensorEventListener
+) {
+  var updateInterval: Long? = null
+    private set
+  private var hasBeenReleased = false
+
+  fun start() {
+    assertSubscriptionIsAlive()
+    if (!_isEnabled) {
+      _isEnabled = true
+      subscribableSensorKernelService.onSubscriptionEnabledChanged(this)
+    }
+  }
+
+  private var _isEnabled = false
+  val isEnabled: Boolean
+    get() = !hasBeenReleased && _isEnabled
```

Comment:
```suggestion
  val isEnabled: Boolean
    get() = !hasBeenReleased && field
```
Or is it too confusing that

this.isEnabled = true
this.isEnabled == false

is possible? (What's idiomatic in Kotlin?)

Reply:
yeah, for now I think a backing field is best to limit scope of this PR, but a refactor is definitely possible to make this a bit cleaner. I do think it's a bit odd to use `field` and something else in the getter.